WebRoot/skins/_base/base4/skin.properties

```diff
@@ -2054,3 +2054,19 @@ ChooserListViewCustomStyleContainer     = @PanelBorder@; @FixBoxModel@;
 MetaPropertyTable       =
 MetaPropertyTableCell   = padding:@SkinWrapperPadding@ @WidgetInternalPadding@; padding-top:0;
 
+#######################################################################
+#   Task List view
+#######################################################################
+
+TaskListViewAddNewTaskIcon                                  = display:inline-block; margin-right:12px;
+TaskListViewNewTaskBannerWrapper                            = @Text-hint@ border-bottom:1px solid @SeparatorColor@; @InactiveCursor@ @FontSize-big@ padding:16px 0 16px 6px;
+TaskListViewHeaderItemLabel                                 = @FontSize-slightly-big@ @Text-light@ padding: 10px 0px 10px 11px;
+TaskReadingPaneRight-ListViewItemCheckboxCell               = padding-left:10px;
+TaskReadingPaneRight-ListViewItem                           = height:80px;
+TaskReadingPaneRight-ListViewItemSubject                    = vertical-align:bottom;
+TaskReadingPaneRight-ListViewItemDate                       = align:right; padding-top:16px;
```

Comment:
That's what I mean, that was used as an attribute and you moved that to css style rule which is not exactly the same.   Let me know if there's some misunderstanding . 

<img width="423" alt="screen shot 2017-06-20 at 6 56 09 pm" src="https://user-images.githubusercontent.com/4294012/27335480-2ae15340-55ea-11e7-843c-cba02903ddf9.png">